docs/index.html

```diff
@@ -427,413 +330,373 @@
             },
             {
-                "quadrant": CLOUD,
-                "ring": TRIAL,
+                "quadrant": DEVOPS,
+                "ring": COMPETENT,
                 "label": "Cloudflare",
                 "active": true,
                 "moved": 0
             },
             {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
-                "label": "AWS Cognito",
+                "quadrant": DEVOPS,
+                "ring": CORE,
+                "label": "AWS",
                 "active": true,
                 "moved": 0
             },
             {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
-                "label": "AWS Cloudfront",
+                "quadrant": DEVOPS,
+                "ring": CORE,
+                "label": "Serverless",
                 "active": true,
                 "moved": 0
-            },
+            },            
             {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
-                "label": "AWS API Gateway",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
-                "label": "AWS VPC",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": CLOUD,
-                "ring": TRIAL,
-                "label": "AWS Web Application Firewall",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
+                "quadrant": DEVOPS,
+                "ring": CORE,
                 "label": "Terraform",
                 "active": true,
                 "moved": 0
             },
             {
-                "quadrant": CLOUD,
-                "ring": TRIAL,
-                "label": "AWS Step Functions",
+                "quadrant": DEVOPS,
+                "ring": CORE,
+                "label": "Kubernetes",
                 "active": true,
                 "moved": 0
             },
+            // End DEVOPS Quadrant
             {
-                "quadrant": CLOUD,
-                "ring": ASSESS,
-                "label": "AWS Cloudformation",
+                "quadrant": DATA,
+                "ring": EXPLORE,
+                "label": "Snowflake",
                 "active": true,
                 "moved": 0
             },
             {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
-                "label": "AWS SES",
+                "quadrant": DATA,
+                "ring": EXPLORE,
+                "label": "PostgreSQL",
                 "active": true,
                 "moved": 0
             },
             {
-                "quadrant": CLOUD,
-                "ring": ADOPT,
-                "label": "AWS ECS/EKS",
+                "quadrant": DATA,
+                "ring": MAINTAIN,
```

Comment:
I'm not sure about this one, we need to discuss it I think. I do want to prevent we add ES to each and every project where it doesn't really add any value